docs/contributing/how-to-release.md

```diff
@@ -2,14 +2,19 @@
 
 Creating a new release requires a few things before getting started:
 
-* Linux or OSX with 7z installed.
-* Windows with HTML Help Workshop installed.
-* Doxygen 1.8.8, and GraphViz installed on both machines.
+* Linux or OSX.
+* Windows with HTML Help Workshop, and Inno Setup installed.
+* 7-Zip, Doxygen 1.8.8, and GraphViz installed on both machines.
```

Comment:
I don't think 7z is a problem, for me the holy grail would be to allow making the releases with a single command from Linux, but this just doesn't seem to be possible (at least without a lot of Wine) and as long as we have to use Windows anyhow, installing any free tools there is not a problem IMHO.